source/neuropod/internal/backend_registration.cc

```diff
@@ -70,7 +70,14 @@ std::vector<BackendLoadSpec> get_default_backend_map()
     const std::vector<FrameworkInfo> frameworks = {
         {"torchscript", "libneuropod_torchscript_backend.so", true, {"1.1.0", "1.2.0", "1.3.0", "1.4.0", "1.5.0"}},
         {"tensorflow", "libneuropod_tensorflow_backend.so", true, {"1.12.0", "1.13.1", "1.14.0", "1.15.0"}},
-        {"python", "libneuropod_pythonbridge_backend.so", false, {"27", "35", "36", "37", "38"}}};
+        {"python", "libneuropod_pythonbridge_backend.so", false, {"2.7", "3.5", "3.6", "3.7", "3.8"}}};
+
+    // Base directory for Neuropod backends
+    std::string neuropod_base_dir = "/usr/local/lib/neuropod";
+    if (auto base_dir = std::getenv("NEUROPOD_BASE_DIR"))
```

Comment:
This is example of whenn "Elvis operator" would be useful:

std::string neuropod_base_dir = std::getenv("NEUROPOD_BASE_DIR") ?: ""/usr/local/lib/neuropod""

Reply:
Yeah, unfortunately it isn't in the C++ standard - just a GCC extension

For anyone who's curious, there's a talk from CppCon2019 that goes over this: ["Non-conforming C++: the Secrets the Committee Is Hiding From You"](https://youtu.be/IAdLwUXRUvg?t=364) (linked to the relevant timestamp in the talk)